### PR TITLE
Add some missing commands and requirements during environment setup

### PR DESCRIPTION
**Pull request by JerryX1110**
Thanks for your neat and thought-provoking work! I've just run the env-setup commands as you provided and found some commands or scripts were lost. Maybe you can have a check about them! : )

### CODE REVIEW

**Review by eliahuhorwitz — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -46,9 +46,12 @@ pip install -r requirements.txt
 ```
 3. Download and extract the dataset
 ```bash
+mkdir datasets
```
Comment (eliahuhorwitz) [inline README.md:49]: I'll add this

File: README.md [hunk 2]
```diff
@@ -46,9 +46,12 @@ pip install -r requirements.txt
...
 cd datasets
+mkdir mvtec3d
+cd mvtec3d
 wget https://www.mydrive.ch/shares/45920/dd1eb345346df066c63b5c95676b961b/download/428824485-1643285832/mvtec_3d_anomaly_detection.tar.xz
-tar -xvf mvtec_3d_anomaly_detection.tar.xz && mv mvtec_3d_anomaly_detection mvtec3d
+tar -xvf mvtec_3d_anomaly_detection.tar.xz
```
Comment (eliahuhorwitz) [inline README.md:54]: I don't think there is a need for these changes, the original command should cover this as the downloaded tar is extracted into the `mvtec_3d_anomaly_detection` dir and the `&& mv mvtec_3d_anomaly_detection mvtec3d` part renames it to `mvtec3d`

File: README.md [hunk 3]
```diff
@@ -127,8 +130,13 @@ As mentioned in the paper, the results reported use the preprocessed dataset. <b
...
 It may take a few hours to run the preprocessing. Results for the preprocessed dataset are reported below. <br>
 
-To run the preprocessing 
+To run the preprocessing, [Open3D](https://github.com/isl-org/Open3D) library is required. 
 ```bash
+# install required packages
+sudo apt install libgl1-mesa-glx
+pip install open3d
+
+# run the preprocessing
```
Comment (eliahuhorwitz) [inline README.md:139]: Added open3d to the requirements.txt file. I'd like to keep the instructions platform-agnostic, so I'd rather not add the sudo install.

**Review by eliahuhorwitz — CHANGES_REQUESTED** [review]
Thanks for the suggestions

File: README.md (hunk 2 above)
Comment (JerryX1110) [reply README.md:54]: As the extraction of mvtec_3d_anomaly_detection.tar.gz would not produce the folder mvtec_3d_anomaly_detection, so mvtec_3d_anomaly_detection folder should be made before extraction.

**Comment by eliahuhorwitz** [comment]
Added your suggestions to the readme. Thanks for pointing them out

**Pull request closed without merging**